### PR TITLE
major updates

### PR DESCRIPTION
**Pull request by geografa**
archive old web
new front page

### CODE REVIEW

**Comment by geografa** [comment]
pdxosgeo
new responsive landing page. Quick and dirty but let's something up before next meeting.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,5 @@
 data/leaflet.sqlite
```
Comment (twelch) [inline .gitignore:1]: If this is old site asset, can this entry be removed?

**Review by twelch — APPROVED** [review]
One small nit, looks good to get started.  Can test further once up.  Thanks @geografa 🎯

File: .gitignore (hunk 1 above)
Comment (geografa) [reply .gitignore:1]: Good call 👍

**Pull request merged**